mllib/src/test/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluatorSuite.scala

```diff
@@ -40,4 +41,51 @@ class MulticlassClassificationEvaluatorSuite
   test("should support all NumericType labels and not support other types") {
     MLTestingUtils.checkNumericTypes(new MulticlassClassificationEvaluator, spark)
   }
+
+  test("Multiclass Classification Evaluator") {
+    val labelAndScores = spark.createDataFrame(Seq(
```

Comment:
Yes. But it maybe needed if the implement change in the future (such as dont call `mllib.MuticlassMetrics`)
In regard to regression metrics, both `ml.RegressionEvaluatorSuite` and `mllib.RegressionMetricsSuite` test the correctness